editor/plugins/script_text_editor.h

```diff
@@ -198,6 +198,8 @@ class ScriptTextEditor : public ScriptEditorBase {
 	virtual void set_syntax_highlighter(Ref<EditorSyntaxHighlighter> p_highlighter) override;
 	void update_toggle_scripts_button();
 
+	CodeTextEditor *get_code_editor();
```

Comment:
Please don't expose this, add indirect methods to `ScriptEditorBase` eg: `set_breakpoint` and `clear_breakpoints`.

Reply:
On `VisualScriptEditor` I don't know how to implement `set_breakpoint`/`clear_breakpoints`, as I don't know how to set breakpoints in VisualScript *(never used it before)*, and on `TextEditor` the `get_breakpoints` is a [dummy function](https://github.com/godotengine/godot/blob/master/editor/plugins/text_editor.cpp#L128). Would it be okay to have blank definitions on those for now?

Reply:
Yeah having dummy / blank methods are fine.

Reply:
Fixed